bee-node/README.md

```diff
@@ -78,3 +78,30 @@ cargo build --release
 cp config.example.toml config.toml
 ../target/release/bee
 ```
+
+# Using Docker
+
+We also provide a `Dockerfile` that allows you to quickly depoly a Bee node.
```

Comment:
the devil hid a typo in this sentence :trollface:

Reply:
It took me way to long to find the error, even after you pointed it out 🙈